.pipelines/.vsts-vhd-windows-automation.yaml

```diff
@@ -41,7 +41,7 @@ steps:
     go install .
     cd ../../..
     echo "go run autonotes to shuffle release note files from Azure Pipelines into the AgentBaker repo structure"
-    go run vhdbuilder/release-notes/autonotes/main.go --build $BUILD_ID --include 2019-containerd,2022-containerd,2022-containerd-gen2,23H2,23H2-gen2
+    go run vhdbuilder/release-notes/autonotes/main.go --build $BUILD_ID --include 2019-containerd,2022-containerd,2022-containerd-gen2,23H2,23H2-gen2,2025-containerd,2025-containerd-gen2
```

Comment:
AKS supports docker in 2019 and containerd is the default for 2022. Now docker is not supported so it does not need to add -containerd. Please reference the name pattern of 23H2